### PR TITLE
Add description and long_description to assembly attributes

### PR DESCRIPTION
**Pull request by ccaio**

### CODE REVIEW

File: sources/assembly-data/ATTR_assembly.types.yaml [hunk 1]
```diff
@@ -89,11 +109,19 @@ attributes:
...
+        description: "Project Psyche (click to view on ENA)"
       prjna707598:
         description: "Squalomix (click to view on ENA)"
       prjna489243:
         description: "Vertebrate Genomes Project (click to view on ENA)"
+      PRJNA955268:
+        description: "Yggdrasil (click to view on ENA)"
   biosample:
+    description: BioSample Identifier
+    long_description: A unique identifier assigned to each entry in the NCBI BioSample database. A BioSample contains descriptions of biological source materials used in experimental assays, such as nucleotide sequencing
```
Comment (rjchallis) [inline sources/assembly-data/ATTR_assembly.types.yaml]: the definition is correct, but not sure that the second sentence will help the LLM

File: sources/assembly-data/ATTR_assembly.types.yaml [hunk 2]
```diff
@@ -116,35 +146,46 @@ attributes:
       count: 10
       scale: log10
   protein_count:
+    description: Protein count
+    long_description: Number of proteins in an assembly derived from annotation
```
Comment (rjchallis) [inline sources/assembly-data/ATTR_assembly.types.yaml]: think there's no need to include "derived from annotation". An alternative would be to say "in an annotated assembly".

File: sources/assembly-data/ATTR_assembly.types.yaml [hunk 3]
```diff
@@ -116,35 +146,46 @@ attributes:
...
     display_group: annotation
     display_level: 2
     display_name: Protein count
     constraint:
       min: 1
     type: integer
   pseudogene_count:
+    description: Pseudogene count
+    long_description: Number of pseudogenes in an assembly derived from annotation
```
Comment (rjchallis) [inline sources/assembly-data/ATTR_assembly.types.yaml]: again no need for "derived from annotation"

File: sources/assembly-data/ATTR_assembly.types.yaml [hunk 4]
```diff
@@ -116,35 +146,46 @@ attributes:
...
     display_group: annotation
     display_level: 2
     display_name: Pseudogene count
     constraint:
       min: 1
     type: integer
   noncoding_gene_count:
+    description: Non-coding gene count
+    long_description: Number of noncoding genes in an organism's genome assembly
```
Comment (rjchallis) [inline sources/assembly-data/ATTR_assembly.types.yaml]: probably omit "organism's"

File: sources/assembly-data/ATTR_assembly.types.yaml [hunk 5]
```diff
@@ -116,35 +146,46 @@ attributes:
...
     constraint:
       min: 1
     type: integer
   sample_sex:
+    description: Sample sex
     display_level: 2
     display_name: Sample sex
     type: keyword
   isolate:
+    description: Isolate
```
Comment (rjchallis) [inline sources/assembly-data/ATTR_assembly.types.yaml]: This should probably be longer so the LLM doesn't read it as a verb

File: sources/assembly-data/ATTR_assembly.types.yaml [hunk 6]
```diff
@@ -116,35 +146,46 @@ attributes:
...
     display_level: 2
     display_name: Isolate
     type: keyword
   assembly_level:
+    description: Highest level of an assembly submission
+    long_description: The highest assembly level for any object in the assembly. The assembly level is a proxy for existence of an assembly of the whole nuclear genome of a taxon. For a taxon, having a value for the assembly_level attribute can be considered the same as having a genome assembly
```
Comment (rjchallis) [inline sources/assembly-data/ATTR_assembly.types.yaml]: ordered from lowest to highest, the assembly level can be "contig", "scaffold"...

proxy for _the_ existence

File: sources/assembly-data/ATTR_assembly.types.yaml [hunk 7]
```diff
@@ -180,6 +256,8 @@ attributes:
     type: long
     units: bases
   contig_n50:
+    description: Contig N50 statistic
+    long_description: Contig N50 is a value for assessment of assembly contiguity. Given a set of contigs, the N50 is defined as the sequence length of the shortest contig at 50% of the total assembly length
```
Comment (rjchallis) [inline sources/assembly-data/ATTR_assembly.types.yaml]: given an _ordered_ set of contigs

File: sources/assembly-data/ATTR_assembly.types.yaml [hunk 8]
```diff
@@ -201,6 +281,8 @@ attributes:
     type: long
     units: bases
   scaffold_n50:
+    description: Scaffold N50 statistic
+    long_description: Scaffold N50 is a value for assessment of assembly contiguity. Given a set of scaffolds, the N50 is defined as the sequence length of the shortest scaffold at 50% of the total assembly length
```
Comment (rjchallis) [inline sources/assembly-data/ATTR_assembly.types.yaml]: given an _ordered_ set of scaffolds

File: sources/assembly-data/ATTR_assembly.types.yaml [hunk 9]
```diff
@@ -222,6 +307,8 @@ attributes:
     type: long
     units: bases
   contig_count:
+    description: Contig count
+    long_description: Total number of contigs in an assembly submission
```
Comment (rjchallis) [inline sources/assembly-data/ATTR_assembly.types.yaml]: omit submission

File: sources/assembly-data/ATTR_assembly.types.yaml [hunk 10]
```diff
@@ -230,6 +317,8 @@ attributes:
     type: long
     units: bases
   scaffold_count:
+    description: Scaffold count
+    long_description: Total number of scaffolds in an assembly submission
```
Comment (rjchallis) [inline sources/assembly-data/ATTR_assembly.types.yaml]: omit submission

File: sources/assembly-data/ATTR_assembly.types.yaml [hunk 11]
```diff
@@ -238,6 +327,8 @@ attributes:
     type: long
     units: bases
   chromosome_count:
+    description: Chromosome count
+    long_description: Total number of chromosome types identified in an assembly submission. The chromosome count is equal as the haploid chromosome number of a taxon when the sequenced individual belongs to the homogametic sex. When the sex of the sequenced sample corresponds to the heterogametic sex of a taxon, then the chromosome count ican be lower, higher or equal the haploid number, depending on the karyotype of the individual
```
Comment (rjchallis) [inline sources/assembly-data/ATTR_assembly.types.yaml]: use of "chromosome types" is confusing here
omit "submission"
is equal _to_ the haploid...
ican -> can
higher or equal _to_ the haploid...

File: sources/assembly-data/ATTR_assembly.types.yaml [hunk 12]
```diff
@@ -246,6 +337,8 @@ attributes:
     type: long
     units: bases
   sequence_count:
+    description: Sequence count
+    long_description: Total number of sequences composing an assembly
```
Comment (rjchallis) [inline sources/assembly-data/ATTR_assembly.types.yaml]: sequences _comprising_ an assembly (or just _in_)

File: sources/assembly-data/ATTR_assembly.types.yaml [hunk 13]
```diff
@@ -273,15 +372,35 @@ attributes:
...
+      6.C: 
+        description: "chromosome-level assemblies with contig N50 >= 1Mb"
+      "6.7": 
+        description: "Assemblies with contig N50 >= 1Mb and scaffold N50 >= 10Mb"
+      5.C: 
+        description: "chromosome-level assemblies with contig N50 >= 100Kb"
+      "5.6": 
+        description: "chromosome-level assemblies with contig N50 >= 100K and scaffold N50 >= 1Mb"
   refseq_category:
+    description: NCBI RefSeq genome classification of a reference or representative genome
```
Comment (rjchallis) [inline sources/assembly-data/ATTR_assembly.types.yaml]: omit "of a reference or representative genome"

File: sources/assembly-data/ATTR_assembly.types.yaml [hunk 14]
```diff
@@ -273,15 +372,35 @@ attributes:
...
+    long_description: RefSeq genome classification of a reference or representative genome in the NCBI  RefSeq database
     display_level: 2
     display_name: RefSeq category
     type: keyword
+    value_metadata:
+      Reference genome:
+        description: A manually selected high-quality genome assembly that NCBI and the community have identified as important as a standard against which other data are compared
```
Comment (rjchallis) [inline sources/assembly-data/ATTR_assembly.types.yaml]: Move to long_description

File: sources/assembly-data/ATTR_assembly.types.yaml [hunk 15]
```diff
@@ -290,6 +409,8 @@ attributes:
     type: 2dp
     units: "%"
   n_percent:
+    description: N percentage
+    long_description: "Percentage of the placeholder for unknown or uncertain nucleotide (N) in a specific position of a sequence or assembly"
```
Comment (rjchallis) [inline sources/assembly-data/ATTR_assembly.types.yaml]: Percentage of unknown or uncertain nucleotides (denoted N) in an assembly

File: sources/assembly-data/ATTR_assembly.types.yaml [hunk 16]
```diff
@@ -355,6 +491,11 @@ attributes:
       enum:
         - primary
         - alternate
-    description: primary or alternate assembly for a taxon
+    description: primary or alternate genome assembly
     display_group: general
     type: keyword
+    value_metadata:
+      primary:
+        description: "The primary assemblies represents the collection of assembled chromosomes, unlocalized and unplaced sequences that, when combined, should represent a non-redundant haploid genome. This excludes any of the alternate locus groups"
```
Comment (rjchallis) [inline sources/assembly-data/ATTR_assembly.types.yaml]: move to long description

File: sources/assembly-data/ATTR_assembly.types.yaml (hunk 1 above)
Comment (ccaio) [reply sources/assembly-data/ATTR_assembly.types.yaml]: Removed second sentence to something useful to EBP projects

**Review by rjchallis — APPROVED** [review]

**Pull request merged**